src/MyElefant.php

```diff
@@ -270,7 +290,26 @@ public function sendSms($campaignId, $content)
                 );
                 return true;
             }
+        } else if ($response->getStatusCode() == 401) {
```

Comment:
by https://www.php-fig.org/psr/psr-7/#33-psrhttpmessageresponseinterface you can use `===` for the status codes